pkg/ddevapp/config_test.go

```diff
@@ -1317,3 +1319,220 @@ func TestDatabaseConfigUpgrade(t *testing.T) {
 		assert.Empty(app.MariaDBVersion)
 	}
 }
+
+// TestConfigMergeStringList verifies that string merges update w/o clobber
```

Comment:
Thanks so much for getting tests together for this!

Reply:
Am I mistaken that this function really only checks for deletions, and thus should have a name indicating that? (Of course, I've already said I'd rather not include tests for features we're not going to include yet.)